message_definitions/v1.0/messages.xml

```diff
@@ -239,12 +239,6 @@
       <field name="south" type="float" unit="m"/>
     </message>
 
-  <!-- Not used, merge with XTEND_RSSI ? -->
-    <message name="RSSI" id="28">
-      <field name="rssi" type="uint8" unit="dB"/>
-      <field name="tx_power" type="uint8" unit="dB"/>
-    </message>
-
```

Comment:
this message is actually used here: https://github.com/paparazzi/paparazzi/blob/master/sw/airborne/modules/multi/rssi.c#L65
so we have to keep it
may be update the comment with `TODO merge with XTEND_RSSI`